Makefile

```diff
@@ -34,22 +34,22 @@ LINUX_BINS = \
 	tracker/tracker
 
 agent/agent:: $(wildcard agent/*.go)
-	$(BUILD_LINUX)
+	$(CROSS_COMPILER)
```

Comment:
We only use a docker container to build when we have to cross compile C (basically just the sqlite driver in build-index and origin). It's really slow and IMO we should avoid it wherever possible.

Reply:
Talked about this offline -- the cross compiling is slow, not the container startup.